packages/agent/app/test-frame.ts

```diff
@@ -33,4 +33,8 @@ export class TestFrame {
     let { args: [message] } = yield this.mailbox.receive();
     return JSON.parse(message.data);
   }
+
+  clear() {
+    this.element.src = 'about:blank';
```

Comment:
Is this syrchronous, is there any event we need to wait on to make sure this teardown is complete?

Reply:
I've made it synchronous for now. We could wait for the element to load the `about:blank` page, but I'm not sure how useful that would be, it should be pretty much instant to load.